### DISCUSSION TITLE
#1 Code Search and Navigation check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#45922] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Welcome to the first Code Search and Navigation check-in of 2023 📆, can’t believe we’re ending the first month of the year.

I posted the [New Code Search and View (Beta) FAQs](https://github.com/orgs/community/discussions/42470) with all that you need to know about the new Code Search and View experience. If you haven’t checked it out yet, you should now! Please post all feedback and feature requests to this [discussion](https://github.com/orgs/community/discussions/38692https://github.com/orgs/community/discussions/38692) so our product team can track feedback all in one place.

## ✨  Explorers 🔍  and Navigators 🚢 

Throttling up with a round of applause 👏  for these awesome community members. 💖 

- [@mhilbush](https://github.com/mhilbush) had no code results while searching a Repo, but they tried the new Code Search beta and got the expected results. Thanks for sharing this with the community. While we are continuously working on improving the tools, [feedback like this](https://github.com/orgs/community/discussions/44388) helps immensely. 👍 
- Thanks [@ijsnow](https://github.com/ijsnow) for sharing valuable feedback for [match highlighting the code](https://github.com/orgs/community/discussions/44809) 💯 
- Great Job by [@xiaohk](https://github.com/xiaohk) for identifying a [bug with Code view](https://github.com/orgs/community/discussions/45272) and sharing it with us. 👏 
- We owe a big thanks to [@jsoref](https://github.com/jsoref) for sharing a ton of great feedback and helping us improve. You are a Super ⭐

## Help us stay categorized! 🤝 

The Code Search and Navigation category is for asking your questions and sharing feedback about the product Code Search and Code Navigation. Please use the appropriate category, or if one doesn’t fit, use the [General](https://github.com/orgs/community/discussions/categories/general) category. This way your post will most likely get seen and responded to by an expert.

With that, it's time to wrap up January with this check-in. I’ll be around next month with another one! ⏭️

**Comment by tonglil** [comment]
I want to preface this with how much I enjoy using the new layout and file tree.

That being said, the reason I am close to opting out of this is the fact that GH is stealing my browser's search, causing an inconsistent experience across pages 😞
1. code editing uses a pretty [lackluster search](https://github.com/dear-github/dear-github/issues/288) (particularly, one I can't prev/next results)
2. code view uses the stolen search
3. every other page on GH uses my native browser search

Things your search cannot do that my browser search does:
- Search from the point on the page, instead from the top
- New symbol highlighting breaks searching for things I highlight, as it steals the highlight
- Regex searching
- Case searching
- Keep search results highlighted when I want to pull up symbol info, as it's one or the other with the stolen search

Lastly, not being able to search the code that's not in view because of lazy rendering is....

Please provide a way to opt-out of hijacking find-on-page, as the rest of the changes are very nice.

Here's all the feedback you got about taking the browser's `ctrl-f`/`cmd-f`.
https://github.com/orgs/community/discussions/39163#discussioncomment-4180000
https://github.com/orgs/community/discussions/38692#discussioncomment-4822054
https://github.com/orgs/community/discussions/38692#discussioncomment-4852544
https://github.com/orgs/community/discussions/38692#discussioncomment-4850531
https://github.com/orgs/community/discussions/38692#discussioncomment-4844811
https://github.com/orgs/community/discussions/38692#discussioncomment-4788011
https://github.com/orgs/community/discussions/38692#discussioncomment-4287526
https://github.com/orgs/community/discussions/38692#discussioncomment-4727683
https://github.com/orgs/community/discussions/38692#discussioncomment-4486125
https://github.com/orgs/community/discussions/38692#discussioncomment-4401867
https://github.com/orgs/community/discussions/38692#discussioncomment-4315743
https://github.com/orgs/community/discussions/38692#discussioncomment-4267069
https://github.com/orgs/community/discussions/38692#discussioncomment-4245103
https://github.com/orgs/community/discussions/38692#discussioncomment-4203123
https://github.com/orgs/community/discussions/38692#discussioncomment-4177932
https://github.com/orgs/community/discussions/38692#discussioncomment-4162358
https://github.com/orgs/community/discussions/38692#discussioncomment-4150192
https://github.com/orgs/community/discussions/38692#discussioncomment-4148305
https://github.com/orgs/community/discussions/38692#discussioncomment-4148070
https://github.com/orgs/community/discussions/38692#discussioncomment-4146291
https://github.com/orgs/community/discussions/38692#discussioncomment-4110600
https://github.com/orgs/community/discussions/38692#discussioncomment-4109714
